lmod_proxy/static/site/js/site.js

```diff
@@ -70,56 +76,52 @@ $(document).ready(function(){
 
   // Submit form handling
   $('#get-membership').click(function(event){
-    post_call({
-      'gradebook': $('#gradebook').val(),
-      'user': $('#user').val(),
-      'section': $('#section').val(),
-      'submit': 'get-membership',
-    });
+    form_data = new FormData();
+    form_data.append('gradebook', $('#gradebook').val());
+    form_data.append('user', $('#user').val());
+    form_data.append('section', $('#section').val());
+    form_data.append('submit', 'get-membership')
+    post_call(form_data)
     event.preventDefault();
   });
 
   $('#get-assignments').click(function(event){
-    post_call({
-      'gradebook': $('#gradebook').val(),
-      'user': $('#user').val(),
-      'section': $('#section').val(),
-      'submit': 'get-assignments',
-    });
+    form_data = new FormData();
+    form_data.append('gradebook', $('#gradebook').val());
+    form_data.append('user', $('#user').val());
+    form_data.append('section', $('#section').val());
+    form_data.append('submit', 'get-assignments')
+    post_call(form_data)
     event.preventDefault();
   });
 
   $('#get-sections').click(function(event){
-    post_call({
-      'gradebook': $('#gradebook').val(),
-      'user': $('#user').val(),
-      'section': $('#section').val(),
-      'submit': 'get-sections',
-    });
+    form_data = new FormData();
+    form_data.append('gradebook', $('#gradebook').val());
+    form_data.append('user', $('#user').val());
+    form_data.append('section', $('#section').val());
+    form_data.append('submit', 'get-sections')
+    post_call(form_data)
     event.preventDefault();
   });
 
```

Comment:
Normal AJAX doesn't allow doing files/multipart data, so we have to build the POST, and then send it raw through an AJAX post as you can't really just add a file to the object.